app/src/lib/stores/commit-status-store.ts

```diff
@@ -778,6 +779,27 @@ export function getCheckDurationInSeconds(
   return 0
 }
 
+/**
+ *  Gets the duration of a check run or job step formatted in minutes and
+ *  seconds.
+ */
+export function getFormattedCheckRunDuration(
+  checkRun: IAPIRefCheckRun | IAPIWorkflowJobStep
+): string {
+  const duration = getCheckDurationInSeconds(checkRun)
+  if (duration < 60) {
+    return `${duration}s`
+  }
+
+  const minutes = Math.floor(duration / 60)
+  const seconds = duration - minutes * 60
+  if (seconds === 0) {
+    return `${minutes}m`
+  }
+
+  return `${minutes}m ${seconds}s`
```

Comment:
Looking at what .com does it seems you'd want to support more than just minutes and hours. I think something like this would match pretty closely

```suggestion
  return moment
    .duration(getCheckDurationInSeconds(checkRun), 'seconds')
    .format('d[d] h[h] m[m] s[s]', { largest: 4 })
```

Reply:
😻